iac-extensions/docker/src/test/java/org/sonar/iac/docker/utils/ArgumentUtilsTest.java

```diff
@@ -81,6 +81,7 @@ void shouldNotResolveVariableOrExpandableStrings(String input) {
     "FROM foo\nARG foo=bar\nLABEL label=${foo:-notbar}",
     "FROM foo\nARG foo=notbar\nARG foo=bar\nLABEL label=$foo",
     "FROM foo\nARG foo=ar\nLABEL label=b$foo",
+    "FROM foo\nENV foo=bar\nLABEL label=$foo",
```

Comment:
Nitpick. Maybe it will be good to add more `ENV` examples here and to other tests in this class?